site/en/docs/extensions/mv2/declare_permissions/index.md

```diff
@@ -31,7 +31,330 @@ Here's an example of the permissions part of a manifest file:
+            href="/docs/extensions/mv2/event_pages">event page</a> or (for hosted apps) a <a
+            href="http://developers.google.com/chrome/apps/docs/background.html">background window</a>.</p>
+      </td>
+    </tr>
+    <tr id="bookmarks">
+      <td><code>"bookmarks"</code></td>
+      <td>Gives your extension access to the <a href="../reference/bookmarks">chrome.bookmarks</a> API.</td>
+    </tr>
+    <tr id="browsingData">
+      <td><code>"browsingData"</code></td>
+      <td>Gives your extension access to the <a href="../reference/browsingData">chrome.browsingData</a> API.</td>
+    </tr>
+    <tr id="certificateProvider">
+      <td><code>"certificateProvider"</code></td>
+      <td>Gives your extension access to the <a href="../reference/certificateProvider">chrome.certificateProvider</a> API.</td>
+    </tr>
+    <tr id="clipboardRead">
+      <td><code>"clipboardRead"</code></td>
+      <td>Required if the extension or app uses <code>document.execCommand('paste')</code>.</td>
+    </tr>
+    <tr id="clipboardWrite">
+      <td><code>"clipboardWrite"</code></td>
+      <td>Indicates the extension or app uses <code>document.execCommand('copy')</code> or
+        <code>document.execCommand('cut')</code>. This permission is <b>required for hosted apps</b>; it's recommended
+        for extensions and packaged apps.</td>
+    </tr>
+    <tr id="contentSettings">
+      <td><code>"contentSettings"</code></td>
+      <td>Gives your extension access to the <a href="../reference/contentSettings">chrome.contentSettings</a> API.</td>
+    </tr>
+    <tr id="contextMenus">
+      <td><code>"contextMenus"</code></td>
+      <td>Gives your extension access to the <a href="../reference/contextMenus">chrome.contextMenus</a> API.</td>
+    </tr>
+    <tr id="cookies">
+      <td><code>"cookies"</code></td>
+      <td>Gives your extension access to the <a href="../reference/cookies">chrome.cookies</a> API.</td>
+    </tr>
+    <tr id="debugger">
+      <td><code>"debugger"</code></td>
+      <td>Gives your extension access to the <a href="../reference/debugger">chrome.debugger</a> API.</td>
+    </tr>
+    <tr id="declarativeContent">
+      <td><code>"declarativeContent"</code></td>
+      <td>Gives your extension access to the <a href="../reference/declarativeContent">chrome.declarativeContent</a> API.</td>
+    </tr>
+    <tr id="declarativeNetRequest">
+      <td><code>"declarativeNetRequest"</code></td>
+      <td>Gives your extension access to the <a href="../reference/declarativeNetRequest">chrome.declarativeNetRequest</a> API.</td>
+    </tr>
+    <tr id="declarativeNetRequestFeedback">
+      <td><code>"declarativeNetRequestFeedback"</code></td>
+      <td>Grants the extension access to events and methods within the <a
+          href="../reference/declarativeNetRequest">chrome.declarativeNetRequest</a> API which return information on declarative
```

Comment:
What is our standard line wrap length? Based on other documents I've seen I thought it was 100 characters, but it looks like this is wrapping at 135.

Reply:
No standard. I don't think this is worth fixing inside inline HTML.